Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -8,6 +8,143 @@
+            "isHiddenWhenLocked": true,
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "225baab0-e4ce-4a45-92de-ddda00795a2d",
+            "version": "KqlParameterItem/1.0",
+            "name": "param_ams_resource",
+            "type": 5,
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.hanaonazure/sapmonitors": true
+              },
+              "additionalResourceOptions": [
+                "value::1"
+              ]
+            },
+            "defaultValue": "value::1"
+          },
+          {
+            "id": "92bd4772-a480-4a10-a48d-fecfd54bb22f",
+            "version": "KqlParameterItem/1.0",
+            "name": "param_providers_list",
+            "type": 1,
+            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"{param_ams_resource}/providerInstances/\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2020-02-07-preview\"},{\"key\":\"pageLimit\",\"value\":\"1000\"}],\"batchDisabled\":false,\"transformers\":null}",
+            "queryType": 12
+          },
+          {
+            "id": "5e3503f6-5f14-4e65-9a5d-f2647affc64d",
+            "version": "KqlParameterItem/1.0",
+            "name": "param_check_sapmon_version",
+            "type": 1,
+            "query": "let sapmonversion = toscalar(\nSapNetweaver_GetSystemInstanceList_CL\n| project SapMonVersion = columnifexists(\"SAPMON_VERSION_s\",'0')\n| summarize SAPMON_VERSION_s = max(SapMonVersion));\n\nlet hasErrors = iif(toint(sapmonversion) >= 2.8, \"0\", \"1\");\n\nprint toscalar(hasErrors);",
+            "isHiddenWhenLocked": true,
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "791b0ca8-eac5-4073-8a9a-330fbce1efb2",
+            "version": "KqlParameterItem/1.0",
+            "name": "param_check_provider_error",
+            "type": 1,
+            "query": "print allProviders=parse_json(tostring(\"{param_providers_list:escapejson}\")).value\r\n| mv-expand allProviders\r\n| extend type=parse_json(allProviders.properties).type\r\n| extend state=parse_json(allProviders.properties).provisioningState\r\n| where type==\"SapNetweaver\"\r\n| where state!=\"Failed\"\r\n| summarize iff(count() > 0,count(), -1)",
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "conditionalVisibility": {
+        "parameterName": "dummy_neverset",
+        "comparison": "isNotEqualTo"
```

Comment:
are you missing a "value" element here?

Reply:
No this is a dummy element added for some calculations which will be never set.